### PR TITLE
State in the manual that NX_CHAR is the default field type

### LINKED ISSUE #892: Explicitly state in the manual that the default field type is NX_CHAR
The NXDL schema states that the default field type is `NX_CHAR` and this should be explicitly stated in the manual. Such a statement is missing from [3.2.1.2. NXDL Field Types and Units](https://manual.nexusformat.org/nxdl-types.html)

### PR DESCRIPTION
**Pull request by benajamin**
State in the manual that NX_CHAR is the default field type, to reflect the existing NXDL behaviour. This change only documents current behaviour and fixes #892.

### CODE REVIEW

**Review by mkoennecke — APPROVED** [review]
Looks good to me

**Review by PeterC-DLS — APPROVED** [review]

File: manual/source/nxdl-types.rst [hunk 1]
```diff
@@ -16,7 +16,8 @@ describe the expected type of data for a NeXus field. These terms are very
 broad. More specific terms are used in actual NeXus data files that describe
 size and array dimensions. In addition to the types in the following table, the
 ``NAPI`` type is defined when one wishes to permit a field
-with any of these data types.
+with any of these data types. The default type ``NX_CHAR`` is applied in cases 
+where a field is defined in an NXDL specification without explicit assignment of a type.
```
Comment (prjemian) [inline manual/source/nxdl-types.rst:20]: where the `type` of a field or attribute

File: nxdlTypes.xsd [hunk 2]
```diff
@@ -460,7 +460,8 @@
 				String arrays cannot be used where only a string is expected 
 				(title, start_time, end_time, ``NX_class`` attribute,...). 
 				Fields or attributes requiring the use of string arrays will be 
-				clearly marked as such (like the ``NXdata`` attribute auxiliary_signals).
+				clearly marked as such (like the ``NXdata`` attribute auxiliary_signals).
```
Comment (prjemian) [inline nxdlTypes.xsd:463]: ``auxiliary_signals``

**Review by prjemian — CHANGES_REQUESTED** [review]
simple changes requested as noted

File: manual/source/nxdl-types.rst (hunk 1 above)
Comment (benajamin) [reply manual/source/nxdl-types.rst]: done in new commit

File: nxdlTypes.xsd (hunk 2 above)
Comment (benajamin) [reply nxdlTypes.xsd]: done in new commit

**Review by prjemian — APPROVED** [review]
Now, this is clearly stated.

**Pull request merged**